ENIGMAsystem/SHELL/Universal_System/instance_system.cpp

```diff
@@ -254,7 +254,7 @@ namespace enigma
     }
 
     if (x < 100000)
-      return x < object_idmax ? objects[x].next ? objects[x].next->inst : NULL : NULL;
+      return size_t(x) < object_idmax ? objects[x].next ? objects[x].next->inst : NULL : NULL;
```

Comment:
Worth pointing out that the negative case in this routine is handled above.

Reply:
Right you are.